src/renderer/containers/Login/LoginContainer.tsx

```diff
@@ -44,4 +71,38 @@ export default class LoginContainer extends React.Component {
             </Background>
         );
     }
-}
\ No newline at end of file
+    private handleChange = (e: React.FormEvent<HTMLInputElement>): void => {
+        this.setState({input: e.currentTarget.value});
+    };
+    
+    private handleSubmit = async (): Promise<void> => {
+
+        const account = await database.account.get("account");
+        console.log(account);
+        if(account!==null){
+            if(account.isCorrectPassword(this.state.input)){
+                this.props.storeAuth(this.state.input);
```

Comment:
NO! You shouldn't store password, you should call account.unlock() and save resulting account into auth state